Public/Src/Utilities/Utilities/RelativePath.cs

```diff
@@ -96,7 +96,7 @@ public static bool TryCreate<T>(StringTable table, T relativePath, out RelativeP
         /// Try to create a RelativePath from a string.
         /// </summary>
         /// <returns>Return the parser result indicating success, or what was wrong with the parsing.</returns>
-        public static ParseResult TryCreate<T>(StringTable table, T relativePath, out RelativePath result, out int characterWithError)
+        public static ParseResult TryCreate<T>(StringTable table, T relativePath, out RelativePath result, out int characterWithError, bool isAbsolute = false)

```

Comment:
> bool isAbsolute = false [](start = 134, length = 24)

The purpose of this isn't obvious when looking at the function signature. Could you please add xml parameter comments for this and the other parameters of this function?

Example:
        /// <param name="pathFormat">Optional override for path format.</param>

Reply:
Sure I will do go ahead and do that.